jenkins/slaves/roles/php/tasks/main.yml

```diff
@@ -62,7 +62,7 @@
   with_items: '{{php_releases}}'
 
 - name: Adjust php memory limit
-  shell: echo "memory_limit = 512M" >> "{{ phpenv_versions_path }}/{{ item.version }}/etc/php.ini"
+  shell: echo "memory_limit = -1" >> "{{ phpenv_versions_path }}/{{ item.version }}/etc/php.ini"
```

Comment:
@jnweiger 

an we also need to set 
date.timezone=UTC

can you please add this as well? THX